Tasks/VsTestV2/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 2,
         "Minor": 141,
-        "Patch": 2
+        "Patch": 3
```

Comment:
master version should be more than release version

Reply:
Make the minor version as 142

Reply:
Ok. Fixed in next commit